webpack.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = _.extend(commonConfig, {
       compress: {
         warnings: false
       },
-      sourceMap: false,
+      sourceMap: true,
```

Comment:
I actually like the idea of including a source map in production. This includes it as a separate file, rather than inline. Browsers only try to pull the source map when developer tools are opened, so it won't incur any addition load time, except for a developer trying to debug the minified library.